cedar-policy-validator/src/lib.rs

```diff
@@ -35,6 +35,7 @@ use cedar_policy_core::ast::{Policy, PolicySet, Template};
 use serde::Serialize;
 use std::collections::HashSet;
 
+pub mod entity_manifest;
```

Comment:
IIRC other experimental features (like partial evaluation & partial validation) are hidden behind flags even in our "private" crates like `cedar-policy-validator`. So you may want to hide this import here behind `"entity-manifest"`

Reply:
thanks! done